CONTRIBUTING.md

```diff
@@ -0,0 +1,283 @@
+- Use the imperative ("add" not "added" or "adds").
+
+#### Types
+
+The following commit types are used in this repository:
+
+| `<type>`       | Repo Label      | Description                                                  |
+| -------------- | --------------- | ------------------------------------------------------------ |
+| `feat`         | `type:feat`     | A new feature                                                |
+| `bug` or `fix` | `type:bug`      | A bug/bug fix (`bug` in issue; `fix` in commit/PR)           |
+| `docs`         | `type:docs`     | Documentation changes                                        |
+| `style`        | -               | Formatting changes that don't affect code functionality      |
+| `refactor`     | `type:refactor` | Code changes that neither fix bugs nor add features          |
+| `test`         | `type:test`     | Add, refactor, modify an EEST library or framework unit test |
+| `chore`        | `type:chore`    | Routine tasks, dependency updates, etc.                      |
+
+#### Scopes
+
+The following scopes are used in this repository:
+
+| `<scope>` | Repo Label         | Description                                    |
+| --------- | ------------------ | ---------------------------------------------- |
+| `fill`    | `scope:fill`       | Changes to `fill` command                      |
+| `execute` | `scope:execute`    | Changes to the `execute` command               |
+| `consume` | `scope:consume`    | Changes to `consume` command suite             |
+| `pytest`  | `scope:pytest`     | Changes that effect all EEST pytest plugins    |
+| `evm`     | `scope:evm`        | Changes to the `evm_transition_tool` package   |
+| `forks`   | `scope:forks`      | Changes to `ethereum_test_forks` package       |
+| `tools`   | `scope:tools`      | Changes to `ethereum_test_tools` package       |
+| `fw`      | `scope:fw`         | Framework changes (evm\|tools\|forks\|pytest)  |
+| `tests`   | `scope:tests`      | Changes to EL client test cases in `./tests`   |
+| `docs`    | `scope:docs`       | Documentation flow changes                     |
+| `ci`      | `scope:ci`         | Continuous Integration changes                 |
+| `gentest` | `scope:gentest`    | Changes to `gentest` CLI command               |
+| `eest`    | `scope:eest`       | Changes to `eest` CLI command                  |
+| `make`    | `scope:make`       | Changes to `eest make` command                 |
+| `tooling` | `scope:tooling`    | Python tools changes (`uv`, `ruff`, `tox`,...) |
+| `deps`    | `scope:deps`       | Updates package dependencies                   |
+
+#### Examples
+
+```console
+feat(eest): add new test generator command
+fix(forks): resolve `Cancun` initialization issue
+docs(fill): describe new command-line args
+refactor(tools): improve code organization in bytecode helpers
+test(pytest): add tests for logging plugin
+chore(deps): update dependency versions
+new(tests): add test cases for EIP-7702
```

Comment:
This is still confusing for me. By the table above, there is no `new` type. Should it be `feat(tests)`?

In general you should put more focus on the work for adding / modifying / fixing `tests`.

Reply:
Maybe put the `feat(tests)` example at the top. And consider adding 1–2 more. What would be the case for `fix(tests)` and `refactor(tests)`?

Reply:
Great suggestion, please see suggestion below: https://github.com/ethereum/execution-spec-tests/pull/1604#discussion_r2090639683

Reply:
Suggestion added.